api/app/controllers/concerns/api/v1/password.rb

```diff
@@ -0,0 +1,16 @@
+module Api
+  module V1
+    # Includes password resetting methods
+    module Password
+      def invalid_token_error
+        render json: { errors: ["Invalid or expired token"] },
+               status: :unprocessable_entity
+      end
+
+      def user_not_found_error
+        render json: { errors: ["No user with that email found"] },
```

Comment:
I think we need to rethink how this resource works... the modeling here doesn't feel right to me (my fault). 

controllers/api/v1/users/reset_password_tokens_controller#create <-- takes email as an argument, always responds with 204 no content, even when email is not found. Whether or not email was found, user sees a message like this: "If a matching account was found an email was sent to user@email.com to allow you to reset your password."

controllers/api/v1/users/password/#update <-- takes token, pw, pw_confirm as arguments and updates user's pw. If successful, we need to somehow return the auth token and user object, exactly like what is returned by the login request (TokensController#create). You may need to move some token response logic into a concern. A user should be logged in after a PW change, so we need to return the same response as with login.

There will also need to be some store/reducer logic (see the auth reducer) to handle this. If a reset password action succeeds, it needs to be reduced similar to how a login access is reduced. This might get tricky, so ask me if you get confused.